src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REM_BOOK = 'REM_BOOK';
+
+let nextId = 0;
+
+export function addBookObject(title, author) {
+  nextId += 1;
+  return {
+    type: ADD_BOOK, id: nextId, title, author,
+  };
+}
```

Comment:
The proper way to set up an Action Creator is to return an object that contains an action type and the payload(response data). You can run into problems with the way you set up your payload since it is not an object but keys for the action object. It is not scalable and difficult to manage. I think it would be best if used the format below:
```
{ type: ADD_BOOK, payload: data }
```

This can be achieved by constructing your data object as a variable. In the future, this will be the response JSON from an API call:
```
const data = {
    id: nextId,
    title,
    author,
}
```

- This groups all the information for a book into its own object that will be passed as a payload to the reducer. The final Action Creator should be similar to this:
```
export function addBookObject(title, author) {
  nextId += 1;
  const data = { id: nextId, title, author }

  return {
    type: ADD_BOOK, payload: data,
  };
}
```
OR
```
  return {
    type: ADD_BOOK, payload: { id: nextId, title, author },
  };
```

Reply:
Implemented 👍